app/build.gradle

```diff
@@ -78,14 +81,17 @@ dependencies {
   // JSON serialization
   compile 'com.google.code.gson:gson:2.6.2'
   // Crash & usage monitoring
-  compile('com.crashlytics.sdk.android:crashlytics:2.5.5@aar') {
+  compile('com.crashlytics.sdk.android:crashlytics:2.6.5@aar') {
     transitive = true;
   }
   // Generate fake data
   compile 'com.github.blocoio:faker:1.2.3'
+  //EventBus
+  compile 'org.greenrobot:eventbus:3.0.0'
   // Testing
   testCompile 'junit:junit:4.12'
   testCompile 'org.mockito:mockito-core:2.0.31-beta'
+  androidTestCompile 'com.android.support.test.uiautomator:uiautomator-v18:2.1.1'
```

Comment:
There were some issues with Home activity tests. Espresso could not correctly perform clicks on the cards and categories, throwing this exception: `Caused by: java.lang.RuntimeException: Action will not be performed because the target view does not match one or more of the following constraints: at least 90 percent of the view's area is displayed to the user.`